scripts/cipublish

```diff
@@ -44,6 +44,12 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                 -f "${DIR}/../docker-compose.test.yml" \
                 run --rm --no-deps \
                 build tile/assembly
+                
+            echo "Building backsplash JAR"
+            docker-compose \
+                -f "${DIR}/../docker-compose.test.yml" \
+                run --rm --no-deps \
+                build backsplash/assembly
```

Comment:
If we add a `.dockeringore` file at the root of the `backsplash` directory with the following contents, it'll bypass shipping the entire `target` directory in the Docker build context. That reduced the assembly JAR build time for me on the first pass to ~5 minutes and subsequent builds to ~33 seconds.

```
target/streams/*
```